docs/src/steady_state_functionality/bifurcation_diagrams.md

```diff
@@ -46,49 +46,67 @@ Finally, we compute our bifurcation diagram using:
 bif_dia = bifurcationdiagram(bprob, PALC(), 2, opts_br; bothside = true)
 nothing # hide
 ```
-(however, in this case these additional settings have no significant effect on the result)
+(In this case, however, these additional settings have no significant effect on the result)
 
-## Bifurcation diagrams with disjoint branches
-Let's consider the previous case, but instead compute the bifurcation diagram over the interval $(2.0,15.0)$:
+## [Bifurcation diagrams with disjoint branches](@id bifurcation_diagrams_disjoint_branches)
+Let's consider the previous case, but instead compute the bifurcation diagram over the interval $(8.0,15.0)$:
 ```@example ex1
-p_span = (2.0, 15.0)
-opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps = 1000)
+p_span = (8.0, 15.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2])
 bif_dia = bifurcationdiagram(bprob, PALC(), 2, opts_br; bothside = true)
-plot(bif_dia; xguide = "k1", yguide = "X")
+plot(bif_dia; ylimit = (0.0, 17.0), xguide = bif_par, yguide = plot_var, branchlabel = "Steady state concentration")
 ```
-Here, in the bistable region, we only see a single branch. The reason is that the continuation algorithm starts at our initial guess (here made at $k1 = 4.0$ for $(X,Y) = (5.0,2.0)$) and tracks the diagram from there. However, with the upper bound set at $k1=15.0$ the bifurcation diagram has a disjoint branch structure, preventing the full diagram from being computed by continuation alone. In this case it could be solved by increasing the bound from $k1=15.0$, however, this is not possible in all cases. In these cases, *deflation* can be used. This is described in the [BifurcationKit documentation](https://bifurcationkit.github.io/BifurcationKitDocs.jl/dev/tutorials/tutorials2/#Snaking-computed-with-deflation).
+Here, in the bistable region, we only see a single branch. The reason is that the continuation algorithm starts at our initial guess (here made at $k1 = 8.0$ for $(X,Y) = (5.0,2.0)$) and tracks the diagram from there. However, with the upper bound set at $k1 = 15.0$ the bifurcation diagram has a disjoint branch structure, preventing the full diagram from being computed by continuation alone. In this case it could be solved by increasing the bound from $k1 = 15.0$, however, this is not possible in all cases. Here we will describe a more general solution, the use of *deflation* to find multiple steady states.
 
+We will not describe the details of how deflation works here (instead, please read [BifurcationKit's documentation](https://bifurcationkit.github.io/BifurcationKitDocs.jl/stable/DeflatedContinuation/)). In practise, in involves first creating a deflation continuation algorithm (`defalg`). This one, in turn, requires us to create [a deflation operator](https://bifurcationkit.github.io/BifurcationKitDocs.jl/stable/library/#BifurcationKit.DeflationOperator) and a perturbation function.
```

Comment:
@rveltz In case you have a more appropriate example(s)/page(s) that you think I should link here instead, I can switch this one out.